src/components/Footer/Footer.tsx

```diff
@@ -38,6 +38,7 @@ const Footer: FC = () => {
   ]
 
   const { t } = useTranslation()
+  const FOOTER_SUPPORT = "FOOTER.SUPPORT"
```

Comment:
? I don't understand why this is done

Reply:
I think it's pointless too, but eslint was making a big deal about re-using the same "FOOTER.SUPPORT" multiple times. Said to assign it to a variable and use that instead. Threw an error because of it. I can undo it or add a line for eslint to by pass it.